code/game/objects/structures/morgue.dm

```diff
@@ -132,7 +132,7 @@ GLOBAL_LIST_EMPTY(bodycontainers) //Let them act as spawnpoints for revenants an
 	playsound(src, 'sound/effects/roll.ogg', 5, 1)
 	playsound(src.loc, 'sound/items/deconstruct.ogg', 50, 1)
 	for(var/atom/movable/AM in connected.loc)
-		if(!AM.anchored || AM == connected)
+		if(!AM.anchored || AM == connected && (istype(AM, /mob) && isliving(AM)))

```

Comment:
isliving is istype(AM, /mob/living), you only need that one, all livings are mobs

Reply:
shiet